client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -1,21 +1,14 @@
 import React, { Component } from 'react';
+import '../../App.scss';
```

Comment:
You don't need this line, since you import same file on line 4, using alias path - import '@src/App.scss';